build/ci-deploy.yml

```diff
@@ -114,7 +118,7 @@ stages:
       resourceGroup: $(DeploymentEnvironmentName)
       testEnvironmentUrl: $(TestEnvironmentUrl)
       imageTag: $(ImageTag)
-      schemaAutomaticUpdatesEnabled: 'tool'
+      schemaAutomaticUpdatesEnabled: 'auto'
```

Comment:
>auto [](start = 38, length = 4)

Can I ask why? I was under impression we always push latest version of schema, and you need to update schema only if your service is living through rollout of new schemas, which shouldn't be the case for CI build.

Reply:
Yes, I'm also confused here. This pipeline is to replace the deployment in ci resource group. With the parameter set to false, all SQL sites cannot find the stored procedures (seems no initilization) and fail to start.

Reply:
[https://github.com/microsoft/healthcare-shared-components/issues/196](https://github.com/microsoft/healthcare-shared-components/issues/196)
Seems like a bug to me, I filed an issue.

---
In reply to: [599859457](https://github.com/microsoft/fhir-server/pull/1768#discussion_r599859457) [](ancestors = 599859457)

Reply:
For CI setting this to `auto` is correct. We don't want to have to manually update the databases for every change and the CI databases are persistent.